### PR TITLE
Bit better docker 

### PR DESCRIPTION
**Pull request by jettify**
Follow up work for https://github.com/aio-libs/aiopg/pull/415

### CODE REVIEW

**Comment by jettify** [comment]
@thehesiod added small fix to your PR here

File: tests/pep492/test_async_await.py [hunk 1]
```diff
@@ -53,6 +54,30 @@
     assert cursor.closed
 
 
+@asyncio.coroutine
+async def test_pool_context_manager_timeout(pg_params, loop):
```
Comment (jettify) [inline tests/pep492/test_async_await.py]: @thehesiod I reworked test a bit, please review

Comment (thehesiod) [reply tests/pep492/test_async_await.py]: hmm, wonder how it worked locally, can't believe I checked it in with the debug code still in there, thanks!  looks good!

Comment (jettify) [reply tests/pep492/test_async_await.py]: Thank! Wanted double check this code.

**Pull request merged**